colin-api/src/colin_api/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.134.0'  # pylint: disable=invalid-name
+__version__ = '2.134.1'  # pylint: disable=invalid-name
```

Comment:
No need to update version in each pr. Will be updated at the time of each release.

Reply:
reverted